File: code_examples/5-Holding_Data/1-Arrays/JavaScript/main.js
/**
 * Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program uses an array
 */

const prompt = require('prompt-sync')()
const studentMarks = new Array(5);
let aSingleMark;

// get length of array
const arrayLength = studentMarks.length;

// input
for (let loopCounter = 0; loopCounter < arrayLength; loopCounter++) {
  aSingleMark = parseInt(prompt("Enter a mark (percentage): "));
  studentMarks[loopCounter] = aSingleMark;
}

console.log();
console.log("Here are the 5 marks:");

for (let loopCounter = 0; loopCounter < arrayLength; loopCounter++) {
  process.stdout.write(studentMarks[loopCounter] + "% ");
}

console.log();
console.log("\nDone.");

File: code_examples/5-Holding_Data/1-Arrays/Python/main.py
#!/usr/bin/env python3
"""
Created by: Mr. Coxall
Created on: Sep 2020
This module uses an array
"""


def main():
    """this function uses an array"""

    student_marks = []

    # input
    for loop_counter in range(0, 5):
        a_single_mark = int(input("Enter a mark (percentage): "))
        student_marks.append(a_single_mark)
    print("")

    print("Here are the 5 marks:")

    for loop_counter in range(0, 5):
        print(f"{student_marks[loop_counter]}% ", end="")

    print("")
    print("\nDone.")


if __name__ == "__main__":
    main()
